connector/src/main/java/com/google/cloud/hive/bigquery/connector/config/HiveBigQueryConfig.java

```diff
@@ -279,6 +293,17 @@ public static HiveBigQueryConfig from(Configuration conf, Map<String, String> ta
             .transform(Boolean::valueOf);
     opts.clusteredFields =
         getAnyOption(CLUSTERED_FIELDS_KEY, conf, tableParameters).transform(s -> s.split(","));
+
+    // Misc
+    opts.flowControlWindowBytes =
```

Comment:
is there need for validity check for the new parameters flow bytes and priorty?

Reply:
AFAICT, the spark connector doesn't validate the value (https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/e0572feb01a7a796f119708a4c4873445ff6c47a/spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java#L413-L415)

I assume maybe the error would be caught further down the line anyways? /cc @davidrabinowitz